Create Boards doc set (#4979)
mattermost/docs: source/boards/accessing-boards.rst

> Accessing Focalboard from Mattermost

Reviewer: Perhaps we can simplify to "Accessing Channel Boards" or similar?

Author: Good suggestion, I like it.

Author: I guess the question is whether it will be accurate to say "Channel Boards" going forward as there will be changes that incorporate all boards (DM, GM, channel) in one place. Maybe "Accessing your Boards" is more future-proof?

Reviewer: You make a fine point! I like your future-proof approach!

> Accessing Focalboard from Mattermost

Author: ```suggestion
Accessing your Boards
```

=====================

Focalboard currently uses channel-based permissions, meaning that only members of the associated channel can access (read/write) the boards for that channel.

You can use this to create private boards:

1. Create or join a Private channel, Group Message, or Direct Message.

> 2. Select the **Focalboard Workspace** icon in the channel header.

Author: ```suggestion
2. Select the **Boards** icon in the channel header.
```

> 2. Select the **Focalboard Workspace** icon in the channel header.

Reviewer: Is it safe to assume that this icon name is also changing to Boards or Boards Workspace?

Author: Great question. Right now it's in flux, kind of like how Playbooks was, and it says "Focalboard Workspace" on community. This will no doubt change but I am leaving it as-is for now.

3. Create a board.

> Only members of channel you created/joined can access the board. However, you can share a read-only version of any board with anyone (including unauthenticated users) who has the generated link.

Author: ```suggestion
Select the Boards icon in the channel header or App Switcher to open the Boards workspace for the current channel, Direct Message, or Group Message - which can only be accessed by members of the channel or message. 

You can share a read-only version of any board with anyone who has the link.
```

> Only members of channel you created/joined can access the board. However, you can use the share board feature to share a read-only board with anyone (incuding unauthenticated users) who has the generated link.

Reviewer: ```suggestion
Only members of channel you created/joined can access the board. However, you can share a read-only version of any board with anyone (including unauthenticated users) who has the generated link.
```

Accessing your Boards
=====================

> Focalboard currently uses channel-based permissions, meaning that only members of the associated channel can access (read/write) the boards for that channel.

Author: ```suggestion
Boards currently uses channel-based permissions, meaning that only members of the associated channel can access (read/write) the boards for that channel.
```
